default-templates/server/index.html

```diff
@@ -11,13 +11,9 @@
 <div class="container">
   <div class="page-header">
     <div class="pull-right">
-      <a href="/register" class="btn btn-primary">
-        Register
-      </a>
+      <a href="/register" class="btn btn-primary">Register</a>
 
-      <a href="/login" class="btn btn-success">
-        Login
-      </a>
+      <a onclick="login()" class="btn btn-success">Login</a>
```

Comment:
Also, this should be changed to a button-element; links with no href has no semantic meaning

Reply:
Myeah they should've been buttons in the first place I suppose. I'll just keep the href.